pro/src/pages/Offers/Offer/Confirmation/__specs__/Confirmation.tracker.spec.jsx

```diff
@@ -149,5 +170,8 @@ describe('confirmation page', () => {
         used: 'ConfirmationPreview',
       }
     )
+    let nbCalls = 1 // page view
+    nbCalls += 1 // click
+    expect(mockLogEvent).toHaveBeenCalledTimes(nbCalls)
```

Comment:
```suggestion
    expect(mockLogEvent).toHaveBeenCalledTimes(2)
```
autant mettre 2 et un commentaire non ?

Reply:
J'ai repris ce qui est fait sur le back avec le compte des appel api.
Ajouter un commentaire pour explicité l'event attendu.